net/src/memory_server.rs

```diff
@@ -40,14 +40,33 @@ fn into_bucket_id(dna_address: &Address, agent_id: &str) -> BucketId {
     format!("{}::{}", dna_address, agent_id)
 }
 
-/// return a unique identifier out of a entry_address and attribute
-pub fn into_meta_id(meta_tuple: &MetaTuple) -> Address {
+/// return a unique identifier out of an entry_address and attribute
+fn into_meta_id(meta_tuple: &MetaTuple) -> Address {
     HashString::from(format!(
         "{}||{}||{}",
         meta_tuple.0, meta_tuple.1, meta_tuple.2
     ))
 }
 
+/// unmerge meta_id into a tuple
+fn _undo_meta_id(meta_id: &Address) -> MetaTuple {
```

Comment:
Unnecessary underscore in front of function name. There are multiple of these in this change